Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -687,9 +689,11 @@ private bool CanSendInGame(string message, IConsoleShell? shell = null, IPlayerS
     }
 
     // ReSharper disable once InconsistentNaming
-    private string SanitizeInGameICMessage(EntityUid source, string message, out string? emoteStr, bool capitalize = true, bool punctuate = false, bool capitalizeTheWordI = true)
+    private string SanitizeInGameICMessage(EntityUid source, string message, out string? emoteStr, bool capitalize = true, bool punctuate = false, bool capitalizeTheWordI = true, bool replaceNetSpeak = true)
     {
         var newMessage = message.Trim();
+        if (replaceNetSpeak)
+            newMessage = SanitizeMessageReplaceWords(newMessage);
         if (capitalize)
```

Comment:
Please don't add dummy args that are never intended to be overwritten, this class is already bloated as is.

Reply:
got it